Source/WebCore/svg/SVGPathElement.cpp

```diff
@@ -194,7 +197,10 @@ ExceptionOr<Ref<SVGPoint>> SVGPathElement::getPointAtLength(float distance) cons
         return Exception { ExceptionCode::InvalidStateError, "The element's path is empty."_s };
 
     // Spec: Clamp distance to [0, length].
```

Comment:
Please move this comment to where the clamping happens:
```
    auto totalLength =  getTotalLength();
    if (totalLength.hasException())
        return totalLength.releaseException();

    // Spec: Clamp distance to [0, length].
    distance = clampTo<float>(distance, 0, totalLength.releaseReturnValue());
```

Reply:
Will update. :-)

Reply:
Done for both.